### ISSUE TITLE
PASU reporting problems

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#1680]
Things wrong in report for PASU:
SUs not in order
duplicate SUs with blank numbers and weights
- [x] check #1541: only bulk should be used

**Comment by aes1884** [comment]
In the second half of the spreadsheet (ceramics tab) where there are duplicates and even triplicates of contexts with a bunch of zeros, I think whatever is wrong is related to the special finds, since the number of times the context is repeated corresponds to the number of ceramic small finds in that context (e.g. there are 2 ceramic small finds from 031, 2 from 035, 3 from 054, etc.)

**Comment by aes1884** [comment]
There are a number of problems in the 'area' tab of the report spreadsheet:
1) bone (ossi) weights are doubled or quadrupled
2) shell (conchiglie) weights are not reported
3) lithics counts and weights are doubled or quadrupled
4) metals counts and weights are quadrupled
5) plaster (intonaco) weights are doubled or quadrupled
6) other counts and weights are doubled/quadrupled in some cases (036, 054) and missing in others (035, 040 - these seem to be entries where there was no count, only weight)
7) some, but not all, of the ceramics counts and weights are doubled (e.g. 042, 050, 054)
8) the metal weights didn't sum at the bottom - it reads "####"

**Comment by aes1884** [comment]
on the ceramics tab, context 039 is missing

**Comment by urapadmin** [comment]
Most of those things are bugs. One thing isn't: 
> the metal weights didn't sum at the bottom - it reads "####"
'###' in Excel means that the column is too small to hold the value. In that case you have to manually widen the column. I can't control that part of the layout. Once you make the column wider the value should appear.

**Comment by urapadmin** [comment]
- [ ] another issue: the rows are not in order. They must be ordered by context.

**Comment by urapadmin** [comment]
### report as of 18.VI.2022
- [ ] only collected material ceramic diag and non-diag of type bulk is now used, eliminating the duplicate rows in the "ceramics" sheet. Those should be okay now. BUT: Make sure that all contexts that should be in the list are actually in the list. Contexts without any bulk ceramics at all are not included.
- [ ]

**Comment by urapadmin** [comment]
@lbestock: Can you please test this? After a few days I have to wrap my head around it from the start again.

**Comment by lbestock** [comment]
Yes but not until some admin stuff is taken care of. It will be hard for me to wrap my head around, too.

**Comment by lbestock** [comment]
Observed in checking the checkboxes above:

There are a few floating things below the totals line on the area tab.
US 016 has a special finds that is ceramic diag. It is right and proper that this not be reported (I believe, still getting my head back in here). However archaeologically I think it's a mistake - 32kg of ceramic in 2k sherds are not a special find.

**Comment by lbestock** [comment]
@urapadmin I think you should take care of the finewares problem before I keep going on this? It's really finicky to look at all these things in the CM and that one is going to be a problem that keeps me seeing others.

**Comment by urapadmin** [comment]
> * (US 016 has a special finds that is ceramic diag. It is right and proper that this not be reported (I believe, still getting my head back in here). However archaeologically I think it's a mistake - 32kg of ceramic in 2k sherds are not a special find, and there are no diagnostic sherds bulk entered for 016.)

That is an archaeological question, indeed. Currently we have decided that small finds are generally not counted because they have been weight in the same bag with the bulk ceramics. Hence only bulk ceramics are part of the ceramics report. That's at least my last memory. That it has its flaws, I agree, but that's for the project to decide.

**Comment by urapadmin** [comment]
### pasu-test 25.06. 2:15 pm EST
fixed:
- There are a few floating things below the totals line on the area tab.
- In fact I note that the ceramics tab has NO tables and fineware at all so that looks to be where the problem is

**Comment by lbestock** [comment]
That's my memory too and it also makes sense. The 016 entry then isn't a question it's just a mistake during the first getting used to of inputting the old data. The others I will test once released.

**Comment by urapadmin** [comment]
it is released (fixing bugs in reports does not need a kiosk release, I only give you a timestamp)

**Comment by lbestock** [comment]
A note for @Petervdom  and @aes1884 - if you correct the 016 entry (which means for right now deleting the one that exists and making a new one that is bulk, not Special Finds), the same thing seems to be an issue for 000. That is also diagnostic ceramic special finds but is definitely in this case not in fact a special find.

**Comment by lbestock** [comment]
I think "other" is pulling from small finds as well as bulk currently. At least SU 036 has 2 other with weight of 2. And I can't figure out what that can be except the bulk other 036H plus the Special Find 036-SF0. Was it intentional to leave SF of types other than ceramic as contributing to the bulk reporting?

**Comment by lbestock** [comment]
This looks good to me. I have checked every cell in area against the recording system and to make sure that the problems noted above were fixed. I have checked that the area and ceramics tabs have the same totals for ceramics. I have spot checked the things taken from the ceramics preprocessing and everything I checked was now correct; it would be a bear to check every cell there, but I will if you want me to. If my test meets your needs, you can close this @urapadmin.

**Comment by urapadmin** [comment]
> SU 036

so this is about altro in "area", I suppose. In area I pull weight and count from all collected materials, no matter what their cm_type is (bulk, sf, sample ...). Except for ceramics. There only bulk is pulled. I guess that is what you are seeing and what makes sense to you (and me).

**Comment by urapadmin** [comment]
I will close this and next is sending it to PASU with instructions on how to update their report.
